### DISCUSSION TITLE
Fix: tv stations in every group

### DISCUSSION THREAD

**Discussion by micha3008** [iptv-org/community#213] [Ideas]
please can you fix the issue that a few channels appear in all countrys
Bloomberg Tv Asia live
blommberg tv asia (720p)
fox movies asia
national geo wild asia 
Bloomberg tv emealive
and other bloomberg stations

please can you check and disalble this channels in the wrong groups

thx

**Comment by BellezaEmporium** [comment]
Bloomberg is on all groups because it is streamed internationally. There's no "bug" whatsoever behind it.

**Comment by micha3008** [comment]
ah reallly. north Korea and many others are little stupid. sorry..
in my opinion. a channel should have one country...

**Comment by BellezaEmporium** [comment]
> ah reallly. north Korea and many others are little stupid. sorry..
> in my opinion. a channel should have one country... 

But ones' channel is meant to be broadcast all around the world. Bloomberg is an example. Another one would be Baby TV. It's European, however each iteration of this channel touch a certain part of the continent.

I can understand that this can give misleading / weird happenings such as Bloomberg being in the NK, but note that apart of this country, it's broadcast everywhere else.

**Comment by micha3008** [comment]
maybe a group world wide channels could be better. you will have clean country lists

**Comment by freearhey** [comment]
If a channel should not be in the playlist of some country, it is enough to fix `broadcast_area` of this channel in the [iptv-org/database](https://github.com/iptv-org/database) and at the next update it will disappear from it.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-feed-description
